SU-HSDO/suhumsci
# HOTFIX - shs-52: revert breakout box color changes from pr #1108

# READY FOR REVIEW

## Summary
This reverts the work found in PR[ #1108](https://github.com/SU-HSDO/suhumsci/pull/1108). It updated the breakout box color, but it was discovered that this breakout box uses a shared class name with other areas of the site.

## Need Review By (Date)
_['asap']_

## Urgency
_['high']_

## Steps to Test
1. Confirm that this reverts to the original CSS code found here: https://github.com/SU-HSDO/suhumsci/pull/1108/files
3. Add a collection, within that collection add a text area. Inside that text area, add an H2 with whatever sample text you want. Save the page. Confirm that it is not using the 'spotlight' color variable for the background.
